README.md

```diff
@@ -10,6 +10,11 @@ A `principal` can have multiple accounts. Each account of a `principal` is ident
 
 The account identified by the subaccount with all bytes set to 0 is the _default account_ of the `principal`.
 
+```
+type SubAccount = nat;
+type Account = record { principal; opt SubAccount; };
```

Comment:
Shouldn't it be
```suggestion
type SubAccount = blob;
type Account = record { of: principal; subaccount : opt SubAccount; };
```
?

Reply:
I like the usage of `of` as field name because it explains that the principal is the owner.

Reply:
type SubAccount = vec nat8;

rather than blob?